community/2017-events/12-contributor-summit/ContribSummitInformation.md

```diff
@@ -18,7 +18,7 @@ Times in CST
 01:00pm - Sessions resume  
 04:30pm - Happy Hour onsite  
 
-View the complete session schedule [here](/kubernetes/community/community/2017-events/schedule).  
+View the complete session schedule [here](/community/community/2017-events/12-contributor-summit/schedule.png).  
```

Comment:
one too many `/community`s in there

Reply:
Should we do relative?

Reply:
How do we not have link checker ... grumble

Reply:
if it's all in the same directory, relative also works. absolute is better if docs are moving around

Reply:
It is in the same dir